app/gui/controller/engine-protocol/src/language_server/tests.rs

```diff
@@ -1,14 +1,40 @@
-use std::future::Future;
+use enso_prelude::*;
+
+use crate::language_server::response;
+use crate::language_server::CapabilityRegistration;
+use crate::language_server::Client;
+use crate::language_server::ContentRoot;
+use crate::language_server::Event;
+use crate::language_server::ExpressionUpdatePayload;
+use crate::language_server::FileAttributes;
+use crate::language_server::FileEdit;
+use crate::language_server::FileEvent;
+use crate::language_server::FileEventKind;
+use crate::language_server::FileSystemObject;
+use crate::language_server::LocalCall;
+use crate::language_server::MethodPointer;
+use crate::language_server::Notification;
+use crate::language_server::Path;
+use crate::language_server::Position;
+use crate::language_server::RegisterOptions;
+use crate::language_server::Result;
+use crate::language_server::Sha3_224;
+use crate::language_server::StackItem;
+use crate::language_server::TextEdit;
+use crate::language_server::TextRange;
+use crate::language_server::Uuid;
+use crate::language_server::VisualisationConfiguration;
+use crate::language_server::API;
```

Comment:
do we need to import so many things? We prefer qualified names, so maybe it would be better to use `language_server::FileEdit` directly instead of importing it? @farmaazon what do u think?

Reply:
Discussed briefly with @wdanilo: let's make a mass import `use::crate::language_server::*`

Reply:
@vitvakatu we agreed on star import ONLY because these are tests. Do not use star import in other, non-test modules please. Just FYI.